setup.py

```diff
@@ -9,15 +9,15 @@
 
 setuptools.setup(
     name="databricks-rocket",
-    version="2.1.0",
+    version="2.2.0",
```

Comment:
Let's set it as a major version update

Reply:
so bump it to 3.0.0?

Reply:
It's backwards compatible, so I would think it should be a minor version update.